eZ/Publish/Core/Persistence/Cache/LocationHandler.php

```diff
@@ -502,4 +502,14 @@ private function getCacheTranslationKey(array $translations = null, bool $useAlw
 
         return implode('|', $translations) . '|' . (int)$useAlwaysAvailable;
     }
+
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
No need to inherit doc, it's inherited by default (I know we have a lot of obsolete PHPDocs like that).